src/CoreBundle/DataAccess/DataAccessCmsMasterPagedefCacheDecorator.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\DataAccess;
+
+use ChameleonSystem\CoreBundle\DataModel\CmsMasterPagdef;
+use ChameleonSystem\CoreBundle\Util\InputFilterUtilInterface;
+use esono\pkgCmsCache\CacheInterface;
+use TGlobal;
+
+class DataAccessCmsMasterPagedefCacheDecorator implements DataAccessCmsMasterPagedefInterface
+{
+    /**
+     * @var DataAccessCmsMasterPagedefInterface
+     */
+    private $subject;
+    /**
+     * @var CacheInterface
+     */
+    private $cache;
+    /**
+     * @var InputFilterUtilInterface
+     */
+    private $inputFilterUtil;
+
+    public function __construct(
+        DataAccessCmsMasterPagedefInterface $subject,
+        InputFilterUtilInterface $inputFilterUtil,
+        CacheInterface $cache
+    ) {
+        $this->subject = $subject;
+        $this->cache = $cache;
+        $this->inputFilterUtil = $inputFilterUtil;
+    }
+
+    public function getPagedefObject(string $pagedef): ?CmsMasterPagdef
+    {
+        $cacheKey = $this->getCacheKeyParameters($pagedef);
+
+        $key = $this->cache->getKey($cacheKey);
```

Comment:
Call this $cacheKey and the above $cacheParameters?

Reply:
ok